config/backend/component-config.local.json

```diff
@@ -0,0 +1,41 @@
+{
+    "loopback-component-explorer": {
+        "mountPath": "/explorer"
+    },
+    "loopback-component-visualizer": {
+        "mountPath": "/visualize"
+    },
+    "loopback-component-model-diagram": {
```

Comment:
I am not too sure this `diagram` is needed for rabbitmq, can we remove it and keep it for later eventually?

Reply:
Same as above